pkg/controller/migmigration/task.go

```diff
@@ -178,9 +173,22 @@ var FailedItinerary = Itinerary{
 		{phase: MigrationFailed},
 		{phase: EnsureStagePodsDeleted, all: HasStagePods},
 		{phase: EnsureAnnotationsDeleted, any: HasPVs | HasISs},
+		{phase: Completed},
+	},
+}
+
+var RollbackItinerary = Itinerary{
+	Name: "Rollback",
+	Steps: []Step{
+		{phase: Rollback},
+		{phase: DeleteBackups},
+		{phase: DeleteRestores},
+		{phase: EnsureStagePodsDeleted, all: HasStagePods},
+		{phase: EnsureAnnotationsDeleted, any: HasPVs | HasISs},
 		{phase: DeleteMigrated},
 		{phase: EnsureMigratedDeleted},
 		{phase: UnQuiesceApplications, all: Quiesce},
+		{phase: RollbackComplete},
```

Comment:
Unless I'm missing the purpose of the `RollbackComplete` phase, it seems like we could drop this and go straight from `UnQuiesce...` to `Completed`.

Reply:
@aufi I still think this should be fixed.

Reply:
Sorry I missed/skipped this before, updated!